.cbmc-batch/jobs/Makefile.aws_hash_table

```diff
@@ -0,0 +1,20 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
This file was copied over from aws-c-common. 

1. We should have an issue to see if we can just use the one from c-common once the subodule thing is working
2. We should maybe note in this file that its a copy of the c-common one

Reply:
I created issue https://github.com/aws/aws-encryption-sdk-c/issues/535 to indicate this should be transfer to AWS C Common.

Reply:
I added a comment on issue #535 to indicate this should be transfer to AWS C Common as well.